### PR TITLE
added rtl direction

### PR DESCRIPTION
**Pull request by saeedrahimi**
Hi there, i've just added rtl direction to this component. there was an issue in rtl direction with `rate` component also when `allowHalf` property used.

This component is used in `ant-design` and i made `ant-design` to support rtl direction. most of ui issues of `rate` component in rtl direction were fixed with css, but `allowHalf` property isn't working properly in rtl direction.
this pr will fix that issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,46 +37,18 @@ npm start
...
-    font-size: ${({ size }) => size}px;
-  }
-`
-
-ReactDOM.render(
-  <StyledRate size="24" />,
-  document.getElementById('root')
-)
+var Rate = require("rc-rate");
+var React = require("react");
```
Comment (afc163) [inline README.md]: import

**Comment by afc163** [comment]
Could you add test case?

**Comment by saeedrahimi** [comment]
> Could you add test case?

image

I can't run tests., could you help on this.
reinstalled `node_modules` it didn't worked.
--
Environment:
OS: Windows 10
node -v:  10.15.0
npm --v: 6.13.1

**Comment by afc163** [comment]
It runs well in my machine:

image

**Pull request closed without merging**